cocos/audio/AudioEngine.cpp

```diff
@@ -82,6 +185,18 @@ bool AudioEngine::lazyInit()
         }
     }
 
+#if CC_TARGET_PLATFORM == CC_PLATFORM_WIN32
+    if (_audioEngineImpl && s_threadPool == nullptr)
+    {
+        s_threadPool = new (std::nothrow) AudioEngineThreadPool(true);
```

Comment:
@WenhaiLin, @zilongshanren 
Why should we `detach` threads only for `Windows` platform?